extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/KuduMetadataRow.java

```diff
@@ -84,4 +95,16 @@ public KuduMetadataRow(final GeoWaveMetadata metadata) {
     return timestamp;
   }
 
+  @Override
+  public void populatePartialRow(PartialRow partialRow, boolean primaryKeyOnly) {
```

Comment:
since the default implementation puts `primaryKeyOnly` as false, let's change this to `includeValue` which defaults to true. Or better yet, maybe we should refactor this to `populatePartialRow` and `populatePartialRowPrimaryKey`